lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,18 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        raise ArgumentError.new("Path can not be used for the application name. Please use --path instead ") if app_name.include?('/')
```

Comment:
How about: `Invalid application name. If you want to set application path, please use --path option`